io/include/pcl/io/hdl_grabber.h

```diff
@@ -152,12 +152,24 @@ namespace pcl
       filterPackets (const boost::asio::ip::address& ipAddress,
                      const unsigned short port = 443);
 
-      /** \brief Allows one to customize the colors used for each of the lasers.
+      /** \brief Allows one to customize the colors used by each laser.
+       * \param[in] color RGB color to set
+       * \param[in] laserNumber Number of laser to set color
        */
       void
       setLaserColorRGB (const pcl::RGB& color,
                         unsigned int laserNumber);
 
+      /** \brief Allows one to customize the colors used for each of the lasers.
+       * \param[in] begin begin iterator of RGB color array
+       * \param[in] end end iterator of RGB color array
+       */
+      template<typename IterT> void
+      setLaserColorRGB( const IterT& begin, const IterT& end )
+      {
+          std::copy( begin, end, laser_rgb_mapping_ );
+      }
```

Comment:
Spacing in the parenthesis in the functions/methods

Reply:
Fixed